src/scripts/choices.js

```diff
@@ -74,19 +65,18 @@ class Choices {
       this.config.renderSelectedChoices = 'auto';
     }
 
-    // Retrieve triggering element (i.e. element with 'data-choice' trigger)
-    const passedElement = isType('String', element)
-      ? document.querySelector(element)
-      : element;
+    let passedElement;
 
-    if (!passedElement) {
-      if (!this.config.silent) {
-        console.error(
-          'Could not find passed element or passed element was of an invalid type',
-        );
-      }
+    if (isType('String', element)) {
+      passedElement = document.querySelector(element);
+    } else if (isElement(element)) {
```

Comment:
If you will use here directly `if (element instanceof HTMLInputElement || element instanceof HTMLSelectElement)` instead of `isElement` helper it will you give well better typed autocomplete in VSCode